docs/data_classes.rst

```diff
@@ -10,10 +10,10 @@ SpectrogramCube
 ---------------
 
 The fundamental data class of the ``sunraster`` package is `~sunraster.SpectrogramCube`.
-It is designed to handle data representing one or more spectrograms of solar regions.
+It is designed to handle data representing one or more spectrograms of some solar regions.
```

Comment:
```suggestion
It is designed to handle data representing one or more spectrograms of solar regions.
```

I think some implies it can't handle spectrograms of other regions of the Sun.